### DISCUSSION TITLE
首次install之后其他中文example都成功运行，英文example指令失败反馈如正文。

### DISCUSSION THREAD

**Discussion by peter65374** [shibing624/pycorrector#327] [Q&A]
安装成功后，其他的几个example都试验通过了，只有readme里面提供的英文的example有如下的信息。 是因为的我的环境是python 3.10，已经不兼容encoding这个参数了么？ 

> >>> import pycorrector
> >>> 
> >>> sent = "what happending? how to speling it, can you gorrect it?"
> >>> corrected_text, details = pycorrector.en_correct(sent)
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "/Users/pt42/opt/miniconda3/lib/python3.10/site-packages/pycorrector/en_spell.py", line 151, in correct
>     self.check_init()
>   File "/Users/pt42/opt/miniconda3/lib/python3.10/site-packages/pycorrector/en_spell.py", line 44, in check_init
>     self._init()
>   File "/Users/pt42/opt/miniconda3/lib/python3.10/site-packages/pycorrector/en_spell.py", line 32, in _init
>     all_word_freq_dict = json.loads(f.read(), encoding="utf-8")
>   File "/Users/pt42/opt/miniconda3/lib/python3.10/json/__init__.py", line 359, in loads
>     return cls(**kw).decode(s)
> TypeError: JSONDecoder.__init__() got an unexpected keyword argument 'encoding'
> >>> print(sent, '=>', corrected_text)
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
> NameError: name 'corrected_text' is not defined. Did you mean: 'corrected_sent'?
> >>> print(details)
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
> NameError: name 'details' is not defined. Did you mean: 'detail'?

**Comment by shibing624** [comment]
用python3.8及以下版本都支持，python3.9和以后版本的encoding变化了，后面我有空兼容下。

**Reply by shibing624** [reply]
fixed.
